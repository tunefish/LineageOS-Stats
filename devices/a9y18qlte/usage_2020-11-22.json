{
  "17.1-20200715-unofficial-tissot": 917,
  "17.1-20200821-unofficial-z3": 1702
}