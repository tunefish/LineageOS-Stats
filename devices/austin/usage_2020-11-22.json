{
  "12.1-20181218-unofficial-austin": 1474,
  "12.1-20190418-unofficial-austin": 12,
  "12.1-20190422-unofficial-austin": 22,
  "12.1-20190524-unofficial-austin": 4,
  "12.1-20190731-unofficial-austin": 5,
  "12.1-20190823-unofficial-austin": 7,
  "12.1-20200423-unofficial-austin": 3,
  "12.1-20200505-unofficial-austin": 44,
  "14.1-20190828-unofficial-austin": 297,
  "4.8-20190418-wechat-fwrenren": 1
}