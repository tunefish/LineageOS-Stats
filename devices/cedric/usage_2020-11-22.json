{
  "1.0.1-cedric": 1,
  "14.1-20170601-unofficial-cedric": 1,
  "14.1-20170706-unofficial-cedric": 3,
  "14.1-20170811-unofficial-cedric": 23,
  "14.1-20171006-unofficial-cedric": 3,
  "14.1-20171018-unofficial-cedric": 7,
  "14.1-20171111_140556-unofficial-cedric": 18,
  "14.1-20180117-unofficial-cedric": 25,
  "14.1-20180204-unofficial-cedric": 4,
  "14.1-20180318-unofficial-cedric": 56,
  "14.1-20180817-unofficial-cedric": 51,
  "14.1-20190207-unofficial-cedric": 1,
  "14.1-20190221-unofficial-cedric": 2,
  "14.1-20190311-unofficial-cedric": 1,
  "14.1-20190407-unofficial-cedric": 2,
  "14.1-20190619-unofficial-cedric": 1,
  "14.1-20190630-unofficial-cedric": 5,
  "14.1-20190706-unofficial-cedric": 1,
  "14.1-20190710-unofficial-cedric": 9,
  "14.1-20190813-unofficial-cedric": 5,
  "14.1-20190912-unofficial-cedric": 23,
  "14.1-20191023-unofficial-cedric": 1,
  "14.1-20191115-unofficial-cedric": 11,
  "14.1-20191228-unofficial-cedric": 1,
  "14.1-20200214-unofficial-cedric": 1,
  "14.1-20200216-unofficial-cedric": 1,
  "14.1-20200304-unofficial-cedric": 1,
  "14.1-20200328-unofficial-cedric": 1,
  "14.1-20200407-unofficial-cedric": 1,
  "14.1-20200424-unofficial-cedric": 3,
  "14.1-20200508-unofficial-cedric": 2,
  "14.1-20200610-unofficial-cedric": 3,
  "14.1-20200708-unofficial-cedric": 2,
  "14.1-20200805-unofficial-cedric": 3,
  "14.1-20200822-unofficial-cedric": 9,
  "14.1-20200910-unofficial-cedric": 1,
  "14.1-20200919-unofficial-cedric": 24,
  "14.1-20201006-unofficial-cedric": 16,
  "14.1-20201106-unofficial-cedric": 21,
  "15.1-20180213-unofficial-cedric": 22,
  "15.1-20180415-unofficial-cedric": 122,
  "15.1-20180422-unofficial-cedric": 13,
  "15.1-20180424-unofficial-cedric": 2,
  "15.1-20180715-unofficial-cedric": 1,
  "15.1-20180723-unofficial-cedric": 5,
  "15.1-20180809-unofficial-cedric": 1,
  "15.1-20180828-unofficial-cedric": 2,
  "15.1-20181016-unofficial-cedric": 2,
  "15.1-20190228-unofficial-cedric": 11,
  "15.1-20190425-unofficial-cedric": 3,
  "15.1-20190916-unofficial-cedric": 2,
  "15.1-20200927-unofficial-cedric": 4,
  "15.1-20201003-unofficial-cedric": 18,
  "15.1-20201012-unofficial-cedric": 5,
  "15.1-20201021-unofficial-cedric": 3,
  "15.1-20201024-unofficial-cedric": 2,
  "15.1-20201105-unofficial-cedric": 1,
  "15.1-20201106-unofficial-cedric": 4,
  "15.1-20201107-unofficial-cedric": 4,
  "15.1-20201114-unofficial-cedric": 17,
  "16.0-20190101-unofficial-cedric": 77,
  "16.0-20190405-unofficial-cedric": 35,
  "16.0-20190522-unofficial-cedric": 28,
  "16.0-20190525-unofficial-cedric": 1,
  "16.0-20190701-unofficial-cedric": 9,
  "16.0-20190709-unofficial-cedric": 43,
  "16.0-20190711-unofficial-cedric": 6,
  "16.0-20190719-unofficial-cedric": 1,
  "16.0-20190817-unofficial-cedric": 2,
  "16.0-20190912-unofficial-cedric": 226,
  "16.0-20191016-unofficial-cedric": 1,
  "16.0-20191211-unofficial-cedric": 1,
  "16.0-20191214-unofficial-cedric": 2,
  "16.0-20200211-unofficial-cedric": 2,
  "16.0-20200311-unofficial-cedric": 3,
  "16.0-20200416-unofficial-cedric": 1,
  "16.0-20200511-unofficial-cedric": 1,
  "16.0-20200522-unofficial-cedric": 1,
  "16.0-20200819-unofficial-cedric": 2,
  "16.0-20200914-unofficial-cedric": 1,
  "16.0-20200915-unofficial-cedric": 1,
  "16.0-20201115-unofficial-cedric": 3,
  "16.0-20201117-unofficial-cedric": 3,
  "17.0-20191214-unofficial-cedric": 162,
  "17.1-20200112-unofficial-cedric": 23,
  "17.1-20200220-unofficial-cedric": 43,
  "17.1-20200319-unofficial-cedric": 3,
  "17.1-20200419-unofficial-cedric": 2,
  "17.1-20200422-unofficial-cedric": 118,
  "17.1-20200520-unofficial-cedric": 3,
  "17.1-20200602-unofficial-cedric": 1,
  "17.1-20200612-unofficial-cedric": 208,
  "17.1-20200623-unofficial-cedric": 1,
  "17.1-20200625-unofficial-cedric": 1,
  "17.1-20200709-unofficial-cedric": 1,
  "17.1-20200715-unofficial-cedric": 2,
  "17.1-20200717-unofficial-cedric": 1,
  "17.1-20200724-unofficial-cedric": 1,
  "17.1-20200731-unofficial-cedric": 1,
  "17.1-20200808-unofficial-cedric": 1,
  "17.1-20200810-unofficial-cedric": 15,
  "17.1-20200811-unofficial-cedric": 2,
  "17.1-20200813-unofficial-cedric": 2,
  "17.1-20200814-unofficial-cedric": 1,
  "17.1-20200815-unofficial-cedric": 2,
  "17.1-20200816-unofficial-cedric": 1,
  "17.1-20200818-unofficial-cedric": 3,
  "17.1-20200819-unofficial-cedric": 2,
  "17.1-20200820-unofficial-cedric": 799,
  "17.1-20200822-unofficial-cedric": 160,
  "17.1-20200903-unofficial-cedric": 1,
  "17.1-20200908-unofficial-cedric": 1,
  "17.1-20200909-unofficial-cedric": 1,
  "17.1-20200910-unofficial-cedric": 1,
  "17.1-20200911-unofficial-cedric": 1,
  "17.1-20200913-unofficial-cedric": 1,
  "17.1-20200915-unofficial-cedric": 4,
  "17.1-20200917-unofficial-cedric": 1,
  "17.1-20200919-unofficial-cedric": 1,
  "17.1-20200920-unofficial-cedric": 6,
  "17.1-20200921-unofficial-cedric": 3,
  "17.1-20200922-unofficial-cedric": 41,
  "17.1-20200927-unofficial-cedric": 1,
  "17.1-20201004-unofficial-cedric": 3,
  "17.1-20201006-unofficial-cedric": 2,
  "17.1-20201016-unofficial-cedric": 5,
  "17.1-20201018-unofficial-cedric": 8,
  "17.1-20201019-unofficial-cedric": 2,
  "17.1-20201020-unofficial-cedric": 3,
  "17.1-20201021-unofficial-cedric": 18,
  "17.1-20201022-unofficial-cedric": 4,
  "17.1-20201023-unofficial-cedric": 2,
  "17.1-20201024-unofficial-cedric": 8,
  "17.1-20201025-unofficial-cedric": 1,
  "17.1-20201026-unofficial-cedric": 15,
  "17.1-20201027-unofficial-cedric": 11,
  "17.1-20201028-unofficial-cedric": 11,
  "17.1-20201029-unofficial-cedric": 6,
  "17.1-20201030-unofficial-cedric": 13,
  "17.1-20201031-unofficial-cedric": 12,
  "17.1-20201101-unofficial-cedric": 14,
  "17.1-20201102-unofficial-cedric": 5,
  "17.1-20201103-unofficial-cedric": 32,
  "17.1-20201104-unofficial-cedric": 1,
  "17.1-20201105-unofficial-cedric": 3,
  "17.1-20201106-unofficial-cedric": 5,
  "17.1-20201107-unofficial-cedric": 9,
  "17.1-20201108-unofficial-cedric": 12,
  "17.1-20201109-unofficial-cedric": 19,
  "17.1-20201110-unofficial-cedric": 16,
  "17.1-20201111-unofficial-cedric": 10,
  "17.1-20201112-unofficial-cedric": 9,
  "17.1-20201113-unofficial-cedric": 11,
  "17.1-20201114-unofficial-cedric": 3,
  "17.1-20201115-unofficial-cedric": 20,
  "17.1-20201119-unofficial-cedric": 9,
  "17.1-20201120-unofficial-cedric": 38,
  "17.1-20201121-unofficial-cedric": 8,
  "aosp_cedric-userdebug 9 pq3a.190801.002 eng.thxbro.20190813.181708 test-keys": 2,
  "qq3a.200705.002": 14,
  "ro.lineage.releasetype=unofficial": 3,
  "rr_cedric-userdebug 10 qp1a.191105.004 eng.root.20191215.073423 test-keys": 17
}