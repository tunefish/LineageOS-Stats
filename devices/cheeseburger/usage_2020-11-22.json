{
  "10.0-20201005-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201007-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201008-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201009-unofficial-snapshot-cheeseburger": 5,
  "10.0-20201011-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201023-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201105-unofficial-snapshot-cheeseburger": 1,
  "10.0-20201116-unofficial-snapshot-cheeseburger": 1,
  "14.1-20170809-unofficial-cheeseburger": 1,
  "14.1-20170826-unofficial-cheeseburger": 1,
  "14.1-20170904-nightly-cheeseburger": 2,
  "14.1-20170906-nightly-cheeseburger": 7,
  "14.1-20170913-unofficial-cheeseburger": 3,
  "14.1-20171002-nightly-cheeseburger": 1,
  "14.1-20171009-nightly-cheeseburger": 2,
  "14.1-20171023-nightly-cheeseburger": 1,
  "14.1-20171106-nightly-cheeseburger": 1,
  "14.1-20171120-nightly-cheeseburger": 6,
  "14.1-20171211-nightly-cheeseburger": 1,
  "14.1-20171214_114902-unofficial-cheeseburger": 2,
  "14.1-20171218-nightly-cheeseburger": 1,
  "14.1-20171225-nightly-cheeseburger": 1,
  "14.1-20180108-nightly-cheeseburger": 2,
  "14.1-20180219-microg-cheeseburger": 1,
  "14.1-20180220-nightly-cheeseburger": 14,
  "14.1-20180409_154130-unofficial-cheeseburger": 1,
  "14.1-20200104-unofficial-cheeseburger": 4,
  "14.1-20200303_092746-unofficial-cheeseburger": 4,
  "14.1-20200830-unofficial-cheeseburger": 2,
  "14.1-20200921-unofficial-cheeseburger": 3,
  "15.1-20180319-nightly-cheeseburger": 1,
  "15.1-20180326-nightly-cheeseburger": 1,
  "15.1-20180402-nightly-cheeseburger": 1,
  "15.1-20180409-nightly-cheeseburger": 1,
  "15.1-20180430-nightly-cheeseburger": 1,
  "15.1-20180514-nightly-cheeseburger": 1,
  "15.1-20180521-nightly-cheeseburger": 2,
  "15.1-20180528-nightly-cheeseburger": 1,
  "15.1-20180611-nightly-cheeseburger": 2,
  "15.1-20180618-nightly-cheeseburger": 2,
  "15.1-20180716-microg-cheeseburger": 2,
  "15.1-20180723-microg-cheeseburger": 2,
  "15.1-20180723-nightly-cheeseburger": 1,
  "15.1-20180730-nightly-cheeseburger": 3,
  "15.1-20180806-nightly-cheeseburger": 4,
  "15.1-20180820-nightly-cheeseburger": 4,
  "15.1-20180831_083810-unofficial-cheeseburger": 3,
  "15.1-20180910-nightly-cheeseburger": 2,
  "15.1-20181015-nightly-cheeseburger": 1,
  "15.1-20181029-nightly-cheeseburger": 2,
  "15.1-20181114-nightly-cheeseburger": 1,
  "15.1-20181116-nightly-cheeseburger": 4,
  "15.1-20181117-nightly-cheeseburger": 2,
  "15.1-20181120-nightly-cheeseburger": 1,
  "15.1-20181203-microg-cheeseburger": 1,
  "15.1-20181214-nightly-cheeseburger": 2,
  "15.1-20181226-nightly-cheeseburger": 1,
  "15.1-20181227-nightly-cheeseburger": 1,
  "15.1-20181230-nightly-cheeseburger": 1,
  "15.1-20190103-nightly-cheeseburger": 1,
  "15.1-20190119-nightly-cheeseburger": 1,
  "15.1-20190120-nightly-cheeseburger": 1,
  "15.1-20190128-microg-cheeseburger": 1,
  "15.1-20190207-nightly-cheeseburger": 1,
  "15.1-20190215-nightly-cheeseburger": 1,
  "15.1-20190221-nightly-cheeseburger": 1,
  "15.1-20190223-nightly-cheeseburger": 1,
  "15.1-20190224-nightly-cheeseburger": 1,
  "15.1-20190225-microg-cheeseburger": 1,
  "15.1-20190225-nightly-cheeseburger": 3,
  "15.1-20200115-unofficial-cheeseburger": 1,
  "15.1-20200619-unofficial-cheeseburger": 7,
  "15.1-20201006-unofficial-cheeseburger": 1,
  "15.1-20201007-unofficial-cheeseburger": 2,
  "15.1-20201008-unofficial-cheeseburger": 2,
  "15.1-20201013-unofficial-cheeseburger": 2,
  "15.1-20201016-unofficial-cheeseburger": 1,
  "15.1-20201019-unofficial-cheeseburger": 1,
  "15.1-20201029-unofficial-cheeseburger": 4,
  "16.0-20181227_153046-unofficial-cheeseburger": 1,
  "16.0-20181228_134325-unofficial-cheeseburger": 1,
  "16.0-20190105_092610-unofficial-cheeseburger": 1,
  "16.0-20190301-nightly-cheeseburger": 1,
  "16.0-20190303-nightly-cheeseburger": 1,
  "16.0-20190304-nightly-cheeseburger": 1,
  "16.0-20190307-nightly-cheeseburger": 2,
  "16.0-20190308-nightly-cheeseburger": 1,
  "16.0-20190311-nightly-cheeseburger": 1,
  "16.0-20190320-nightly-cheeseburger": 1,
  "16.0-20190405-nightly-cheeseburger": 1,
  "16.0-20190417-nightly-cheeseburger": 2,
  "16.0-20190425-nightly-cheeseburger": 1,
  "16.0-20190428-nightly-cheeseburger": 1,
  "16.0-20190430-nightly-cheeseburger": 2,
  "16.0-20190503-nightly-cheeseburger": 2,
  "16.0-20190504-nightly-cheeseburger": 2,
  "16.0-20190509-nightly-cheeseburger": 1,
  "16.0-20190513-microg-cheeseburger": 1,
  "16.0-20190518-nightly-cheeseburger": 1,
  "16.0-20190519-nightly-cheeseburger": 1,
  "16.0-20190523-nightly-cheeseburger": 1,
  "16.0-20190524-nightly-cheeseburger": 1,
  "16.0-20190529-nightly-cheeseburger": 1,
  "16.0-20190531-nightly-cheeseburger": 2,
  "16.0-20190608-nightly-cheeseburger": 3,
  "16.0-20190612-nightly-cheeseburger": 1,
  "16.0-20190616-nightly-cheeseburger": 1,
  "16.0-20190617-nightly-cheeseburger": 2,
  "16.0-20190623-nightly-cheeseburger": 1,
  "16.0-20190624-nightly-cheeseburger": 1,
  "16.0-20190701-microg-cheeseburger": 1,
  "16.0-20190702-nightly-cheeseburger": 1,
  "16.0-20190706-nightly-cheeseburger": 1,
  "16.0-20190708-nightly-cheeseburger": 1,
  "16.0-20190712-nightly-cheeseburger": 1,
  "16.0-20190713-nightly-cheeseburger": 1,
  "16.0-20190713_161241-unofficial-cheeseburger": 4,
  "16.0-20190714-nightly-cheeseburger": 1,
  "16.0-20190716-nightly-cheeseburger": 2,
  "16.0-20190718-nightly-cheeseburger": 1,
  "16.0-20190722-nightly-cheeseburger": 1,
  "16.0-20190723-nightly-cheeseburger": 1,
  "16.0-20190727-nightly-cheeseburger": 1,
  "16.0-20190728-nightly-cheeseburger": 2,
  "16.0-20190803-nightly-cheeseburger": 1,
  "16.0-20190805-microg-cheeseburger": 1,
  "16.0-20190810-nightly-cheeseburger": 1,
  "16.0-20190827-nightly-cheeseburger": 1,
  "16.0-20190913-nightly-cheeseburger": 2,
  "16.0-20190924-nightly-cheeseburger": 1,
  "16.0-20190925-nightly-cheeseburger": 1,
  "16.0-20190929-nightly-cheeseburger": 2,
  "16.0-20191003-nightly-cheeseburger": 1,
  "16.0-20191007-microg-cheeseburger": 1,
  "16.0-20191014-microg-cheeseburger": 1,
  "16.0-20191014-nightly-cheeseburger": 3,
  "16.0-20191016-nightly-cheeseburger": 1,
  "16.0-20191017-nightly-cheeseburger": 2,
  "16.0-20191017-unofficial-cheeseburger": 1,
  "16.0-20191018-nightly-cheeseburger": 1,
  "16.0-20191020-nightly-cheeseburger": 1,
  "16.0-20191022-nightly-cheeseburger": 3,
  "16.0-20191026-nightly-cheeseburger": 1,
  "16.0-20191027-nightly-cheeseburger": 1,
  "16.0-20191028-nightly-cheeseburger": 1,
  "16.0-20191110-nightly-cheeseburger": 1,
  "16.0-20191117-nightly-cheeseburger": 1,
  "16.0-20191122-nightly-cheeseburger": 1,
  "16.0-20191126-nightly-cheeseburger": 1,
  "16.0-20191130-nightly-cheeseburger": 1,
  "16.0-20191203-nightly-cheeseburger": 3,
  "16.0-20191207-nightly-cheeseburger": 1,
  "16.0-20191209-microg-cheeseburger": 1,
  "16.0-20191210-nightly-cheeseburger": 1,
  "16.0-20191213-nightly-cheeseburger": 1,
  "16.0-20191214-nightly-cheeseburger": 1,
  "16.0-20191217-nightly-cheeseburger": 1,
  "16.0-20191218-nightly-cheeseburger": 2,
  "16.0-20191220-nightly-cheeseburger": 1,
  "16.0-20191223-nightly-cheeseburger": 1,
  "16.0-20191229-nightly-cheeseburger": 1,
  "16.0-20191230-nightly-cheeseburger": 2,
  "16.0-20191231-nightly-cheeseburger": 1,
  "16.0-20200101-nightly-cheeseburger": 1,
  "16.0-20200103-nightly-cheeseburger": 2,
  "16.0-20200107-nightly-cheeseburger": 1,
  "16.0-20200109-nightly-cheeseburger": 1,
  "16.0-20200111-nightly-cheeseburger": 1,
  "16.0-20200115-nightly-cheeseburger": 1,
  "16.0-20200123-nightly-cheeseburger": 5,
  "16.0-20200130-nightly-cheeseburger": 2,
  "16.0-20200204-nightly-cheeseburger": 1,
  "16.0-20200211-nightly-cheeseburger": 3,
  "16.0-20200214-nightly-cheeseburger": 1,
  "16.0-20200217-nightly-cheeseburger": 1,
  "16.0-20200220-nightly-cheeseburger": 1,
  "16.0-20200224-microg-cheeseburger": 1,
  "16.0-20200224-nightly-cheeseburger": 3,
  "16.0-20200304-nightly-cheeseburger": 1,
  "16.0-20200308-nightly-cheeseburger": 1,
  "16.0-20200309-nightly-cheeseburger": 3,
  "16.0-20200319-nightly-cheeseburger": 1,
  "16.0-20200325-nightly-cheeseburger": 4,
  "16.0-20200331-microg-cheeseburger": 1,
  "16.0-20200406-nightly-cheeseburger": 2,
  "16.0-20200408-microg-cheeseburger": 1,
  "16.0-20200413-nightly-cheeseburger": 15,
  "16.0-20200519-unofficial-microg-signed-cheeseburger": 2,
  "16.0-20200521-microg-cheeseburger": 1,
  "16.0-20200528-microg-cheeseburger": 1,
  "16.0-20200602-nightly-cheeseburger": 2,
  "16.0-20200608-nightly-cheeseburger": 3,
  "16.0-20200612-microg-cheeseburger": 1,
  "16.0-20200615-nightly-cheeseburger": 6,
  "16.0-20200619-microg-cheeseburger": 1,
  "16.0-20200622-nightly-cheeseburger": 2,
  "16.0-20200629-nightly-cheeseburger": 1,
  "16.0-20200706-nightly-cheeseburger": 7,
  "16.0-20200710-microg-cheeseburger": 1,
  "16.0-20200713-nightly-cheeseburger": 3,
  "16.0-20200717-microg-cheeseburger": 1,
  "16.0-20200720-nightly-cheeseburger": 6,
  "16.0-20200724-microg-cheeseburger": 1,
  "16.0-20200727-nightly-cheeseburger": 5,
  "16.0-20200728-unofficial-cheeseburger": 5,
  "16.0-20200803-nightly-cheeseburger": 18,
  "16.0-20200807-microg-cheeseburger": 2,
  "16.0-20200815-unofficial-cheeseburger": 9,
  "16.0-20200816-microg-cheeseburger": 8,
  "16.0-20200824-nightly-cheeseburger": 7,
  "16.0-20200831-nightly-cheeseburger": 13,
  "16.0-20200904-unofficial-cheeseburger": 12,
  "16.0-20200907-nightly-cheeseburger": 8,
  "16.0-20200914-nightly-cheeseburger": 17,
  "16.0-20200918-unofficial-cheeseburger": 5,
  "16.0-20200921-nightly-cheeseburger": 50,
  "16.0-20200921-unofficial-cheeseburger": 1,
  "16.0-20200922-unofficial-cheeseburger": 2,
  "16.0-20200923-unofficial-cheeseburger": 1,
  "16.0-20200924-unofficial-cheeseburger": 1,
  "16.0-20200925-unofficial-cheeseburger": 1,
  "16.0-20200927-microg-cheeseburger": 16,
  "16.0-20200928-nightly-cheeseburger": 59,
  "16.0-20200928-unofficial-cheeseburger": 1,
  "16.0-20200929-unofficial-cheeseburger": 1,
  "16.0-20201005-nightly-cheeseburger": 62,
  "16.0-20201012-nightly-cheeseburger": 76,
  "16.0-20201019-nightly-cheeseburger": 58,
  "16.0-20201020-microg-cheeseburger": 42,
  "16.0-20201021-unofficial-cheeseburger": 4,
  "16.0-20201026-nightly-cheeseburger": 268,
  "16.0-20201026-unofficial-cheeseburger": 1,
  "17.1-20200117_153227-unofficial-cheeseburger": 81,
  "17.1-20200401-unofficial-cheeseburger": 3,
  "17.1-20200713-unofficial-cheeseburger": 2,
  "17.1-20200803-unofficial-cheeseburger": 1,
  "17.1-20200825-unofficial-cheeseburger": 1,
  "17.1-20200913-unofficial-cheeseburger": 3,
  "17.1-20200916-unofficial-cheeseburger": 10,
  "17.1-20200919-microg-signed-cheeseburger": 18,
  "17.1-20200920-unofficial-cheeseburger": 6,
  "17.1-20200924-unofficial-cheeseburger": 1,
  "17.1-20200927-unofficial-cheeseburger": 1,
  "17.1-20200928-unofficial-cheeseburger": 11,
  "17.1-20201001-unofficial-cheeseburger": 25,
  "17.1-20201004-unofficial-cheeseburger": 1,
  "17.1-20201008-microg-signed-cheeseburger": 10,
  "17.1-20201011-unofficial-cheeseburger": 1,
  "17.1-20201013-unofficial-cheeseburger": 16,
  "17.1-20201020-unofficial-cheeseburger": 3,
  "17.1-20201024-microg-signed-cheeseburger": 8,
  "17.1-20201024-unofficial-cheeseburger": 1,
  "17.1-20201026-unofficial-cheeseburger": 9,
  "17.1-20201031-unofficial-cheeseburger": 4,
  "17.1-20201101-microg-signed-cheeseburger": 3,
  "17.1-20201102-unofficial-cheeseburger": 4,
  "17.1-20201103-unofficial-cheeseburger": 3,
  "17.1-20201104-unofficial-cheeseburger": 5,
  "17.1-20201105-unofficial-cheeseburger": 3,
  "17.1-20201106-microg-signed-cheeseburger": 6,
  "17.1-20201107-unofficial-cheeseburger": 9,
  "17.1-20201108-unofficial-cheeseburger": 3,
  "17.1-20201109-microg-signed-cheeseburger": 15,
  "17.1-20201110-nightly-cheeseburger": 263,
  "17.1-20201113-unofficial-cheeseburger": 1,
  "17.1-20201115-microg-signed-cheeseburger": 5,
  "17.1-20201116-nightly-cheeseburger": 199,
  "17.1-20201117-unofficial-cheeseburger": 1,
  "17.1-20201118-unofficial-cheeseburger": 2,
  "17.1-20201120-unofficial-cheeseburger": 1,
  "18.0-20201006-unofficial-cheeseburger": 1,
  "18.0-20201007-unofficial-cheeseburger": 28,
  "18.0-20201008-unofficial-cheeseburger": 1,
  "18.0-20201009-unofficial-cheeseburger": 42,
  "18.0-20201013-unofficial-cheeseburger": 43,
  "18.0-20201014-unofficial-cheeseburger": 2,
  "18.0-20201017-unofficial-cheeseburger": 22,
  "18.0-20201019-unofficial-cheeseburger": 18,
  "18.0-20201022-unofficial-cheeseburger": 33,
  "18.0-20201027-unofficial-cheeseburger": 33,
  "18.0-20201031-unofficial-cheeseburger": 5,
  "18.0-20201102-unofficial-cheeseburger": 22,
  "18.0-20201105-unofficial-cheeseburger": 36,
  "18.0-20201110-unofficial-cheeseburger": 32,
  "18.0-20201116-unofficial-cheeseburger": 1,
  "18.0-20201117-unofficial-cheeseburger": 9,
  "18.0-20201118-unofficial-cheeseburger": 1,
  "18.0-20201119-unofficial-cheeseburger": 16,
  "18.0-20201120-unofficial-cheeseburger": 1,
  "njh47f release-keys": 30,
  "oneplus5-userdebug 7.1.2 njh47f 4e7963bf6d test-keys": 21,
  "oneplus5-userdebug 7.1.2 njh47f c076374176 test-keys": 4
}