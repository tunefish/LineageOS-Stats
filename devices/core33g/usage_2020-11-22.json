{
  "13.0-20160827-unofficial-core33g": 5,
  "14.1-20170409-unofficial-core33g": 8,
  "14.1-20170820-unofficial": 83,
  "14.1-20170820-unofficial-core33g": 5,
  "14.1-20180615-unofficial-core33g": 163,
  "14.1-20180804-unofficial-core33g": 12,
  "14.1-20190421-unofficial-core33g": 2,
  "14.1-20190523-unofficial-core33g": 18,
  "14.1-20190609-unofficial-core33g": 3357,
  "14.1-20200802-unofficial-core33g": 1,
  "15.1-20180223-unofficial-core33g": 35,
  "15.1-20180708-unofficial-core33g": 67,
  "15.1-20180709-unofficial-core33g": 1,
  "15.1-20180711-unofficial-grandneove3g": 57,
  "17.1-20200715-unofficial-tissot": 6,
  "17.1-20200716-unofficial-herolte": 6,
  "17.1-20200821-unofficial-z3": 37
}