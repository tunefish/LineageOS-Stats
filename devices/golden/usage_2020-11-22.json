{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 75,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 6,
  "13.0-20170310-unofficial-golden": 12,
  "13.0-20170328-unofficial-golden": 604,
  "13.0-20170420-unofficial-golden": 33,
  "14.1-20170122-unofficial-golden": 201,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 6,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170507-unofficial-golden": 15,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 363,
  "14.1-20170610-unofficial-golden": 6,
  "14.1-20170628-unofficial-golden": 603,
  "14.1-20170724-unofficial-golden": 35,
  "14.1-20170817-unofficial-golden": 25,
  "14.1-20170920-unofficial-golden": 7,
  "14.1-20171011-unofficial-golden": 57,
  "14.1-20171215_192311-official-golden": 118,
  "14.1-20171215_192311-unofficial-golden": 604,
  "14.1-20180109_002536-unofficial-golden": 138,
  "14.1-20181014-unofficial-golden": 2,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}